components/husky_a300/networking/husky_a300_peplink_wifi_client.mdx

```diff
@@ -0,0 +1,49 @@
+#### Connecting Husky to a Wi-Fi Network  {#wifi-client-setup}
+
+To get Husky AMP connected to your local Wi-Fi network, follow the steps below.
+
+1. Connect to Husky AMP over an [Ethernet connection](#ethernet-connection) or through
+   its [Wi-Fi Access Point](#wifi-ap-connection).
+1. In a web browser, open the configuration page for Husky AMP's router: https://192.168.131.51
+   - Username: `admin`
+   - Default password: `Clearpath1`
+1. Click on either the "Wi-Fi WAN on 2.4 GHz" or "Wi-Fi WAN on 5GHz" link, depending on

```

Comment:
Change all "text-here" content to `text-here`. This change is consistent with other Husky AMP content.